### DISCUSSION TITLE
以大模型为基础构建短视频工作流

### DISCUSSION THREAD

**Discussion by hydexi0122** [InternLM/Tutorial#194] [project]
班级：3班
性质：实验性项目
技术：书生浦语，书生灵笔，SD，MJ，ChatGPT，SVD或Pika
目的：一直想做一个短视频工作流，尽可能自动化、高效且效果较好，那就挤时间尝试吧，成功哪有这么容易
成员：自己，理想很丰满现实很骨感，因为不能保证最后效果所以一人一组

**Comment by YinzhenWan** [comment]
你好，我一直也想做这个，我之前在公司的时候，有考虑过，做一个自动化的短视频工作流

**Comment by alcholiclg** [comment]
对这个很感兴趣，虽然还是个小白但想马住学习尝试一下。
